packaging/click/launch.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+SCRIPT_SRC=osmscout-server.conf
+SCRIPT_DST=/home/phablet/.config/upstart/osmscout-server.conf
+
+if [[ ! -f "${SCRIPT_DST}" ]]; then
+	cp "${SCRIPT_SRC}" "${SCRIPT_DST}"
+	start osmscout-server
+fi
```

Comment:
do you still need reboot to make it start on the background with this? is `start` waiting for service to start or moves on immediately?

Reply:
hmm, I think it is better to wait for it to start before proceeding. if it is possible that it can wait. then the service will start and later GUI will contact it

Reply:
that got resolved in the newer version

Reply:
We have the choice. By default it will wait. If we didn't want it to wait we could add `--no-wait`.

Reply:
messed up, this is not resolved if we don't wait.

Reply:
Let's wait then - it should be better that way, I think. sorry for many messages